infra/vars.tf

```diff
@@ -14,3 +14,9 @@ variable "INBOUND_KINESIS" {
   type        = string
   default     = "kinesis-hudi-inbound"
 }
+
+variable "OUTPUT_FORMAT" {
```

Comment:
I don't think this variable will be used in the vars.tf file. The idea is to pass it to a module for the flink application, that way we can cleanly create two kda applications for the benchmark:

module "kda_json" {
  source = "./modules/our_kda_module"
  output_format = "json"
 ...
}

module "kda_hudi" {
  source = "./modules/our_kda_module"
  output_format = "hudi"
 ...
}

Reply:
sure!! thats the idea but as i've commented, in the meantime i've included as a bar until the module is included, once the module is included (already included), we have just to remove from bars and point to module input instead of vars